1-js/07-object-properties/02-property-accessors/article.md

```diff
@@ -87,29 +87,29 @@ let user = {
 */!*
 };
 
-// set fullName is executed with the given value.
+// همراه با مقدار داده شده اجرا می‌شود set fullName
 user.fullName = "Alice Cooper";
 
 alert(user.name); // Alice
 alert(user.surname); // Cooper
 ```
 
-As the result, we have a "virtual" property `fullName`. It is readable and writable.
+در نتیجه، ما یک ویژگی «مجازیِ» `fullName` داریم. هم قابل خواندن است و هم قابل نوشتن.
 
-## Accessor descriptors
+## توصیف‌کننده‌های دسترسی
 
-Descriptors for accessor properties are different from those for data properties.
+توصیف‌کننده‌های ویژگی‌های دسترسی نسبت به توصیف‌کننده‌های ویژگی‌های داده‌ای تفاوت دارند.
 
-For accessor properties, there is no `value` or `writable`, but instead there are `get` and `set` functions.
+برای ویژگی‌های دسترسی، `value` یا `writable` وجود ندارد اما به جای آن‌ها تابع‌های `get` و `set` وجود دارد.
 
-That is, an accessor descriptor may have:
+یعنی اینکه یک توصیف‌کننده دسترسی ممکن است این‌ها را داشته باشد:
 
-- **`get`** -- a function without arguments, that works when a property is read,
-- **`set`** -- a function with one argument, that is called when the property is set,
-- **`enumerable`** -- same as for data properties,
-- **`configurable`** -- same as for data properties.
+- متد **`get`** -- تابعی بدون آرگومان، زمانی که ویژگی‌ای خوانده شود کار می‌کند،
+- متد **`set`** -- تابعی با یک آرگومان، زمانی که ویژگی تنظیم می‌شود فراخوانی می‌شود،
+- ویژگی **`enumerable`** -- مشابه به ویژگی‌های داده‌ای،
+- ویژگی **`configurable`** -- مشابه به ویژگی‌های داده‌ای،
```

Comment:
The lines 107 to 110 started from left, because there were English words in the beginning, and they didn't look nice.  
I added Persian words in the beginning, so the lines start from right.